<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentMenu</class>
 <widget class="QDialog" name="studentMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Menu</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>My Attributes</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonViewProjects">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View Projects</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Student Number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textNumber">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ability to lead the team.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Leadership:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_2">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your major CGPA average.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>CGPA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_3">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The grade you want to achieve for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Grade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prefer to do most of the coding for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Prefers to code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_5">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prefer to do most of the documentation for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Documentation:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="a12">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Public speaking ability.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Speaking skills:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How likely you are to procrastinate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Procrasination:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How well you work independently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Independency:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How vocal you are.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Vocal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How good you think your organization skills are.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Organization:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your level of experience.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Experience:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Desired Attributes</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>16</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many hours are you going to dedicate per week?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Hours:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_10">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The grade you want to achieve for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Grade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How well you work independently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Independency:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_15">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your major CGPA average.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>CGPA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_16">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ability to lead the team.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Leadership:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_17">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prefer to do most of the coding for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Prefers to code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_18">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prefer to do most of the documentation for the project.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Documentation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_19">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Public speaking ability.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Speaking skills:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_20">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How likely you are to procrastinate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Procrasination:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_21">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How many hours are you going to dedicate per week?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Hours:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_22">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How vocal you are.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Vocal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_23">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your level of experience.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Experience:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA_24">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How good you think your organization skills are.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Organization:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pa1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
